src/libraries.sh

```diff
@@ -266,12 +276,12 @@ Usage:
 
 =item --all
 
-Read Makefile in current/original directory, pick library requirements up and
-import them all.
+Read $BEAKERLIB_DIR/metadata.yaml or ./Makefile, pickup the library requirements
+up and import them all.
 
 =item LIBRARY
 
-Must have 'component/library' format. Identifies the library to import.
+Must have 'component[/path]' format. Identifies the library to import.
```

Comment:
This syntax is not clear to me, what does 'component[/path]' mean (especially the '[/path]' part)?

Reply:
It's meant to be optional part. Maybe we can simply say it is a path. The complicated thing to describer is the fact the searched path may include `Library` in the second component of the path.

Reply:
It must be complicated because I don't understand it even with your explanation :)
Can you please give an example of the rlImport call with this syntax so I can make sense of it?

Reply:
e.g. the library distribution/epel is searched for in `./distribution/epel` but also in `./distribution/Library/epel`

Originally, the naming convention `COMPONENT/LIBRARY` was desing exactly for th purpose where the `Library` path component was inserted in between. However, this need is now gone and we can have the library located directly in the root of the upstream git repo, meaning the library may be reference only by `epel`. That actual means the library would be searched for in `./epel` as well as `./epel/Library`. And a because of the the upstream libraries structure the first part of the name is the "COMPONENT" as a git repo and the rest is simply a patch to the `lib.sh`.
So I cannot simply say the name coresponds to the path equally.

Reply:
Try to read it in context of the previous paragraph which describe the paths for search.

Reply:
I see, shouldn't then the usage be `(component|path)` ?

Reply:
Not really but if it is more digestible we could use it.